docs/GettingStarted.md

```diff
@@ -132,6 +132,28 @@ Before downloading tracing on your application, install the Datadog Agent. The R
 
 ### Quickstart for Rails applications
 
+#### Rails Auto Instrument all Integrations
+
+1. Add the `ddtrace` gem to your Gemfile:
+
+    ```ruby
+    source 'https://rubygems.org'
+    gem 'ddtrace'
```

Comment:
Does this pattern work with Rails?
```suggestion
    gem 'ddtrace', require: 'ddtrace/auto_instrument'
```
This will technically import `auto_instrument` when needed. I'm not sure if the import order would work correctly, but I think it's something worth exploring.

When trying this change, make sure to remove the changes added to `config/application.rb`, as they hopefully won't be needed.

Reply:
i'll test this, this makes life easier i think

Reply:
👍 yup this works, updated